lib/core-events/src/index.ts

```diff
@@ -1,4 +1,20 @@
-module.exports = {
+export interface CoreEvents {
+  CHANNEL_CREATED: string;
+  GET_CURRENT_STORY: string;
+  SET_CURRENT_STORY: string;
+  GET_STORIES: string;
+  SET_STORIES: string;
+  SELECT_STORY: string;
+  APPLY_SHORTCUT: string;
+  STORY_ADDED: string;
+  FORCE_RE_RENDER: string;
+  REGISTER_SUBSCRIPTION: string;
+  STORY_RENDERED: string;
+  STORY_ERRORED: string;
+  STORY_THREW_EXCEPTION: string;
+}
+
+const events: CoreEvents = {
```

Comment:
Do you think we can throw away the above interface if we use `enum` ? 
Both solutions offer auto-completion in js and ts

I did 
```typescript
enum events {
  CHANNEL_CREATED = 'channelCreated',
  GET_CURRENT_STORY = 'getCurrentStory',
  SET_CURRENT_STORY = 'setCurrentStory',
  GET_STORIES = 'getStories',
  SET_STORIES = 'setStories',
  SELECT_STORY = 'selectStory',
  APPLY_SHORTCUT = 'applyShortcut',
  STORY_ADDED = 'storyAdded',
  FORCE_RE_RENDER = 'forceReRender',
  REGISTER_SUBSCRIPTION = 'registerSubscription',
  STORY_RENDERED = 'storyRendered',
  STORY_ERRORED = 'storyErrored',
  STORY_THREW_EXCEPTION = 'storyThrewException',
}

export default events;
```
and it compiled to 
```typescript
Object.defineProperty(exports, "__esModule", { value: true });
var events;
(function (events) {
    events["CHANNEL_CREATED"] = "channelCreated";
    events["GET_CURRENT_STORY"] = "getCurrentStory";
    events["SET_CURRENT_STORY"] = "setCurrentStory";
    events["GET_STORIES"] = "getStories";
    events["SET_STORIES"] = "setStories";
    events["SELECT_STORY"] = "selectStory";
    events["APPLY_SHORTCUT"] = "applyShortcut";
    events["STORY_ADDED"] = "storyAdded";
    events["FORCE_RE_RENDER"] = "forceReRender";
    events["REGISTER_SUBSCRIPTION"] = "registerSubscription";
    events["STORY_RENDERED"] = "storyRendered";
    events["STORY_ERRORED"] = "storyErrored";
    events["STORY_THREW_EXCEPTION"] = "storyThrewException";
})(events || (events = {}));
exports.default = events;
```

before it was 
```typescript
var events = {
    CHANNEL_CREATED: 'channelCreated',
    GET_CURRENT_STORY: 'getCurrentStory',
    SET_CURRENT_STORY: 'setCurrentStory',
    GET_STORIES: 'getStories',
    SET_STORIES: 'setStories',
    SELECT_STORY: 'selectStory',
    APPLY_SHORTCUT: 'applyShortcut',
    STORY_ADDED: 'storyAdded',
    FORCE_RE_RENDER: 'forceReRender',
    REGISTER_SUBSCRIPTION: 'registerSubscription',
    STORY_RENDERED: 'storyRendered',
    STORY_ERRORED: 'storyErrored',
    STORY_THREW_EXCEPTION: 'storyThrewException',
};
exports.default = events;
```

Reply:
Works for me – I'll give that change a shot.